.prettierrc

```diff
@@ -3,4 +3,4 @@
   "singleQuote": true,
   "tabWidth": 2,
   "useTabs": false
-}
\ No newline at end of file
+}
```

Comment:
I was going to suggest `"trailingComma": "es5"` but apparently it's now default in prettier@^2.x.

Makes diffs way better, objects and arrays especially.

Reply:
Still takes some getting used to to seeing a trailing comma on multiline objects, arrays though 🙈

Reply:
You'll learn to love it in time, when commits that change the last line in an object or array actually show a single line of diff instead of 2 😄